mistletoe/base_renderer.py

```diff
@@ -143,6 +143,9 @@ def render_raw_text(self, token) -> str:
         """
         return token.content
 
+    def render_html_attributes(self, token: block_token) -> str:
```

Comment:
After moving the new stuff out of the "base" code, also move this method to `HTMLAttributesRenderer()`, amend its type hint and return `''` from it?

Reply:
OK.